config.py

```diff
@@ -46,6 +54,42 @@ def bot_token() -> str:
     return _settings.get("bot_token")
 
 
+def redis_host() -> str:
+    """
+    :return: The redis host, as a string.
+    """
+    return _settings.get("redis_host")
+
+
+def redis_port() -> str:
+    """
+    :return: The redis port, as an integer.
+    """
+    return _settings.get("redis_port")
+
+
+def redis_username() -> str:
+    """
+    :return: Username used for redis connection, as a string.
+    """
+    return _settings.get("redis_username") or None
+
+
+def redis_password() -> str:
+    """
+    :return: Password used for redis connection, as a string.
+    """
+    return _settings.get("redis_password") or None
+
+
+def redis_key_const() -> str:
+    return _settings.get("redis_key_const")
+
+
+def roulette_unmute_rate() -> int:
+    return _settings.get("roulette_unmute_rate")
+
+
```

Comment:
The return type here for these fields is `Optional[str]` or `Optional[int]` respectively (they can return `None`)

Note to self: I should add a static analysis checker to this project eventually. Or just change it to TypeScript...